collect_app/src/main/res/values/strings.xml

```diff
@@ -174,7 +174,7 @@
     <string name="save_as_error">Saved name must not be blank!</string>
     <string name="edit_prompt">Edit Prompt</string>
     <string name="selected_google_account_text">Google account</string>
-    <string name="server_platform_odk_aggregate">ODK Aggregate</string>
+    <string name="server_platform_odk_aggregate">ODK</string>
```

Comment:
I think this can now be untranslated? Or at least the resource name should change to `server_platform_odk` or `server_platform_default`.

Reply:
This needs to be addressed before this PR is merged so translators get the right thing. Coming back to it, I have a preference for adding an ODK string to `untranslated.xml` for this instead of sending a translatable string.

Reply:
Ach my bad for dismissing. WIll change this!